tcpb/utils.py

```diff
@@ -135,6 +185,9 @@ def job_output_to_atomic_result(
                 "orbb_occupations": jo_dict.get("orbb_occupations"),
                 "excited_state_energies": jo_dict.get("energy"),
                 "cis_transition_dipoles": jo_dict.get("cis_transition_dipoles"),
+                "mm_gradient": jo_dict.get("mmatom_gradient"),
+                "dipole_moment": jo_dict.get("dipoles")[3],
+                "dipole_vector": jo_dict.get("dipoles")[:3],
```

Comment:
Dipole moment: https://github.com/MolSSI/QCElemental/blob/ec9d754414824e129729f477e6077a14d859dc85/qcelemental/models/results.py#L69

Any chance these values match a field found in the `results.py` in `QCElemental`?